src/pages/main_layout.cr

```diff
@@ -21,35 +21,109 @@ abstract class MainLayout
+            div class: "mx-auto md:px-24 lg:px-32" do
+              h1 class: "leading-normal text-blue-lightest font-light text-3xl md:text-4xl" do
+                text "Build "
+                span "lightning fast", class: "whitespace-no-wrap text-white italic mr-1 border-b-2 border-teal-light"
+                text " web apps with "
+                span "fewer bugs", class: "whitespace-no-wrap text-white italic border-b-2 border-teal-light"
+              end
 
-            para class: "opacity-75 sm:px-10 leading-loose mt-10 text-lg sm:text-xl" do
-              text <<-TEXT
-              A Crystal web framework that catches bugs for you, runs
-              incredibly fast, and helps you write code that lasts.
-              TEXT
-            end
+              para class: "text-blue-lighter lg:px-24 leading-loose mt-10 text-lg sm:text-xl" do
+                text <<-TEXT
+                Lucky is a web framework written in Crystal. It helps you
+                work quickly, catch bugs at compile time, and deliver
+                blazing fast responses.
+                TEXT
+              end
 
-            div class: "my-10" do
-              link "View on GitHub", "https://github.com/luckyframework/lucky", class: "btn btn--blue w-full sm:w-auto sm:mr-5"
-              link "Get Started", Guides::GettingStarted::Installing, class: "btn w-full sm:w-auto mt-6 sm:mt-0"
+              div class: "my-10 md:mt-10 md:mb-8" do
+                link "View on GitHub", "https://github.com/luckyframework/lucky", class: "btn btn--blue w-full sm:w-auto sm:mr-5"
+                link "Get Started", Guides::GettingStarted::Installing, class: "btn w-full sm:w-auto mt-6 sm:mt-0"
+              end
             end
           end
         end
-        render_main_content
+        render_feature_grid
+        render_freeform_text
       end
     end
   end
 
-  private def render_main_content
-    div class: "container mx-auto px-6" do
+  private def render_feature_grid
+    div class: "container text-center md:text-left mx-auto px-6 py-5 bg-white md:bg-transparent border-b border-grey-light md:border-none" do
       mount Shared::FlashMessages.new(@context.flash)
-      # TODO: Add real page content later
-      # content
+      div class: "flex flex-col md:flex-row md:bg-white mx-auto md:rounded-lg md:shadow md:-mt-12 md:px-5 md:py-12" do
+        content_block "Say goodbye to slow", <<-TEXT
+        A simple hello world response is returned in 0.1ms. Rendering
+        complex JSON usually takes only a few milliseconds.
+        TEXT
+
+        content_block "Batteries included", <<-TEXT
+        Authentication, asset management, CORS, database ORM, and more can
+        all be included when creating a new Lucky project.
+        TEXT
+
+        content_block "Fewer bugs. More joy", <<-TEXT
+        Instead of finding bugs in QA or in production, Lucky is designed to
+        catch as many bugs as possible at compile time.
+        TEXT
```

Comment:
Would love your thoughts on this @edwardloveall @jwoertink

Reply:
Love it. 👍